README.md

```diff
@@ -260,7 +433,12 @@ If you need to clear your database, run the following commands:
   }
 }
 ```
- - To create a user upvote or downvote and retrieve updated top 5 results:
+
+##### `createVote` Mutation
```

Comment:
There is a mistake in here somewhere with the backticks and it's not displaying the code blocks properly in MD. I think it might have to do w/ the `createVote` but I am having a hard time pinpointing it without trying it with a preview.